### DISCUSSION TITLE
BOM?

### DISCUSSION THREAD

**Discussion by Eebigdog** [gusmanb/logicanalyzer#64] [General]
Hello, is a BOM available - esp for the connectors used and where to get them?

Thanks

**Comment by gusmanb** [comment]
Hi.

You can get the BOM from the KiCad project, it can export it as every component has its value and packaging.

Regarding where to get the connectors, you can get all of those from any provider like Mouser, Element14 or Digikey, I bought them from AliExpress, the links are these:

Female:
https://es.aliexpress.com/item/1005001288367127.html
Male:
https://es.aliexpress.com/item/4000081601899.html

The TXU0104 (if you want to build the level shifter) can also be bought from AliExpress but when it comes to IC's I prefer always to go to Mouser:
https://www.mouser.es/ProductDetail/595-TXU0104PWR

Hope it helps.

**Comment by Eebigdog** [comment]
Excellent! Thanks for the PN’s.BR,GrantSent from my iPhoneOn Jun 16, 2023, at 3:48 PM, Agustín Gimenez Bernad ***@***.***> wrote:﻿
Hi.
You can get the BOM from the KiCad project, it can export it as every component has its value and packaging.
Regarding where to get the connectors, you can get all of those from any provider like Mouser, Element14 or Digikey, I bought them from AliExpress, the links are these:
Female:
https://es.aliexpress.com/item/1005001288367127.html
Male:
https://es.aliexpress.com/item/4000081601899.html
The TXU0104 (if you want to build the level shifter) can also be bought from AliExpress but when it comes to IC's I prefer always to go to Mouser:
https://www.mouser.es/ProductDetail/595-TXU0104PWR
Hope it helps.
